p2pserver/message/utils/msg_router.go

```diff
@@ -57,6 +59,11 @@ func (this *MessageRouter) init(p2p p2p.P2P) {
 	this.stopConsCh = make(chan bool)
 	this.p2p = p2p
 
+	this.msgWP = &msgWorkerPool{
+		maxWorkerCount : 256 * config.DefConfig.P2PNode.MaxConnInBoundForSingleIP,
```

Comment:
too much?

Reply:
256 is the goroutine numbers for single inbound peer. In my opion,  goroutine  is minor executing unit, 256 is not too much for single peer.